scripts/add_existing_baseyear.py

```diff
@@ -518,6 +518,25 @@ def add_heating_capacities_installed_before_baseyear(
                 build_year=int(grouping_year),
                 lifetime=costs.at[f"{name_type} gas boiler", "lifetime"],
             )
+            # add biomass boilers
+            n.madd(
+                "Link",
+                nodes,
+                suffix=f" {name} biomass boiler-{grouping_year}",
+                bus0=nodes + " solid biomass",
```

Comment:
```suggestion
                bus0=spatial.biomass.nodes,
```
since biomass can also be copperplated in the `config.yaml`